### ISSUE TITLE
Bug any time trying to use GF to replace

### ISSUE THREAD

**Issue by Jimrnelson** [VFPX/GoFish#41]
## ℹ Computer information

- VFP version: 5815
- GoFish version:  5.1.008
- Last GoFish version w/o bug (if available):

## 📝 Provide detailed reproduction steps (if any)

It has been a while since I have used GF to do replacements.  To my surprise, I now get an error (which I have replicated multiple times)

The bug ("object is not contained in a form") occurs whenever I click on the checkbox in the GF results grid to select rows that are to be replaced.

I note that the current date on GF is 2022/11/22 and my last GF replacement was a few weeks before that, so this is the first time that I have tried replacing using the new version.

## 📷 Screenshots

_Are there any useful screenshots? WinKey+Shift+S and then just paste them directly into the form_

![image](https://user-images.githubusercontent.com/81179335/215516443-04294bd6-52cf-4a41-bcc6-86ff2b7c307c.png)

![image](https://user-images.githubusercontent.com/81179335/215516503-7f4c1146-87bb-4151-b8bb-34419de80664.png)

**Comment by Jimrnelson** [comment]
I have learned more on this.

Somewhere in the conversion process, GoFishSearchEngine was converted from a VCX to PRG.  

This code was left in, which of course will fail in a PRG.

![image](https://user-images.githubusercontent.com/81179335/215591235-533cbfc5-f765-469e-b62d-17216c2d938e.png)

**Comment by Jimrnelson** [comment]
Well, that's not quite right.

Am pretty sure that changing Thisform to This is all that is needed

**Comment by lscheffler** [comment]
You are right. Somehow this THISFORM was not changed with the other places of cCR_StoreLocal in this class.
